src/sjcl-ecdsa-signdeterministic.js

```diff
@@ -0,0 +1,119 @@
+'use strict';
+var sjcl = require('sjcl');
+/* eslint-disable new-cap */
+
+sjcl.ecc.ecdsa.secretKey.prototype.generateK =
+function(hash, hashObject) {
+
+  var curve = this._curve;
+  var qlen = sjcl.bitArray.bitLength(curve.r.toBits());
+
+  /* Utility functions */
+  /* used to generate k and v */
+  function repeat(str, times) {
+    return (new Array(times + 1)).join(str);
+  }
+  function bits2int(bits) {
+
+    var blen = sjcl.bitArray.bitLength(bits);
+
+    if (blen > qlen) {
```

Comment:
I think I mentioned this before, but if `blen is > qlen` then `qlen < blen` is always true right? That second branch will never execute here.